src/main/java/uk/gov/hmcts/reform/bulkscanccdeventhandler/transformation/services/CaseValidator.java

```diff
@@ -0,0 +1,20 @@
+package uk.gov.hmcts.reform.bulkscanccdeventhandler.transformation.services;
+
+import com.google.common.base.Strings;
```

Comment:
Either guava's or spring's `StringUtil`. Just a note ;)

Reply:
this is guava

Reply:
yes it is. like i said - just a note. if possible to achieve everything with one library - there's no need of extra one just to do some "nice to have" things if it's appropriate to call like that. once again - just a note. either way it's good